docs/metrics.md

```diff
@@ -5,8 +5,11 @@
 * [Introduction](#introduction)
 * [System metrics](#system-metrics)
 * [Formatting](#formatting)
+* [Metric Types](#metric-types)
+* [Central metrics yaml file definition](#central-metrics-yaml-file-definition)
 * [Custom Metrics API](#custom-metrics-api)
-* [Logging the custom metrics](#logging-the-custom-metrics)
+* [Log custom metrics](#log-custom-metrics)
+* [Metrics YAML Parsing and Metrics API example](#Metrics-YAML-File-Parsing-and-Metrics-API-Custom-Handler-Example)
 
```

Comment:
Should add a known issues section: for example the first inference is now significantly slower